packages/template-retail-react-app/app/pages/checkout/index.test.js

```diff
@@ -511,63 +359,190 @@ test.skip('Can proceed through checkout as registered customer', async () => {
     // Place the order
     user.click(placeOrderBtn)
 
+    // Should now be on our mocked confirmation route/page
+    expect(await screen.findByText(/success/i)).toBeInTheDocument()
```

Comment:
From here onward, the file changes diverge a lot. It's better if you view them using a split view instead.